modules/machineconfig-garbage-collect-viewing.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * post_installation_configuration/machine-configuration-tasks.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="machineconfig-garbage-collect-viewing_{context}"]
+= Viewing rendered machine configs
+
+You can view a list of rendered machine configs by using the `oc adm prune renderedmachineconfigs` command with the `list` subcommand. For example, the command in the following procedure would list all rendered machine configs for the `worker` machine config pool:   
+
+.Procedure
+
+* List the rendered machine configs:
```

Comment:
```suggestion
* List the rendered machine configs by running the following command:
```

Reply:
I can never remember if we use _by running the following command_ or not!

Reply:
afaik, for a procedure step that introduces a command we need to provide an explicit instruction to "run" or "enter" the command.